### PR TITLE
Updating README to reflect correct installation steps

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,23 +45,22 @@ Activate the virtual environment:
...
 Migrate the database to the latest schema:
 
-- `python ./manage.py migrate`
+- `python manage.py migrate`
 
-Mock data can be loaded into your dev site with the following command
+By default, Django sets the site domain to `example.com`, but the mock data needs the domain to be `localhost:8000`. Run the following command to update the site domain automatically
 
-- `python ./manage.py loaddata network-api/networkapi/fixtures/test_data.json`
+- `python manage.py update_site_domain`
```
Comment (mmmavis) [inline README.md]: Not directly related to this PR but I got errors after running `python manage.py update_site_domain`  (fresh install)

```bash
(venv) mavisou:~/Development/network-test/network-api[master*]$ python manage.py update_site_domain
Traceback (most recent call last):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: django_site

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/Users/mavisou/Development/network-test/network-api/networkapi/management/commands/update_site_domain.py", line 9, in handle
    site = Site.objects.get(id=1)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 379, in get
    num = len(clone)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 238, in __len__
    self._fetch_all()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 1087, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 835, in execute_sql
    cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: django_site
```

Comment (mmmavis) [reply README.md]: Not sure if the fix is to update the command in README or fix the script? @gvn @gideonthomas

File: README.md [hunk 2]
```diff
@@ -43,23 +45,22 @@ Activate the virtual environment:
...
 
-By default, Django sets the site domain to `example.com`, but the mock data needs the domain to be `localhost:8000`. Run the following command to update the site domain automatically
+Mock data can be loaded into your dev site with the following command
 
-- `python ./manage.py update_site_domain`
+- `python manage.py loaddata networkapi/fixtures/test_data.json`
```
Comment (mmmavis) [inline README.md]:
```bash
(venv) mavisou:~/Development/network-test/network-api[master*]$ python manage.py loaddata networkapi/fixtures/test_data.json
Traceback (most recent call last):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/contrib/contenttypes/models.py", line 20, in get_by_natural_key
    ct = self._cache[self.db][(app_label, model)]
KeyError: 'default'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
sqlite3.OperationalError: no such table: django_content_type

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/json.py", line 80, in Deserializer
    for obj in PythonDeserializer(objects, **options):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/python.py", line 183, in Deserializer
    obj = base.build_instance(Model, data, db)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/base.py", line 223, in build_instance
    obj.pk = Model._default_manager.db_manager(db).get_by_natural_key(*natural_key).pk
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/contrib/contenttypes/models.py", line 22, in get_by_natural_key
    ct = self.get(app_label=app_label, model=model)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 379, in get
    num = len(clone)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 238, in __len__
    self._fetch_all()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 1087, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 835, in execute_sql
    cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.db.utils.OperationalError: no such table: django_content_type

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 22, in <module>
    execute_from_command_line(sys.argv)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 367, in execute_from_command_line
    utility.execute()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/__init__.py", line 359, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/base.py", line 294, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/base.py", line 345, in execute
    output = self.handle(*args, **options)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 64, in handle
    self.loaddata(fixture_labels)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 104, in loaddata
    self.load_label(fixture_label)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/management/commands/loaddata.py", line 161, in load_label
    for obj in objects:
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/json.py", line 86, in Deserializer
    six.reraise(DeserializationError, DeserializationError(e), sys.exc_info()[2])
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/json.py", line 80, in Deserializer
    for obj in PythonDeserializer(objects, **options):
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/python.py", line 183, in Deserializer
    obj = base.build_instance(Model, data, db)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/core/serializers/base.py", line 223, in build_instance
    obj.pk = Model._default_manager.db_manager(db).get_by_natural_key(*natural_key).pk
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/contrib/contenttypes/models.py", line 22, in get_by_natural_key
    ct = self.get(app_label=app_label, model=model)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 379, in get
    num = len(clone)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 238, in __len__
    self._fetch_all()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 1087, in _fetch_all
    self._result_cache = list(self.iterator())
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/query.py", line 54, in __iter__
    results = compiler.execute_sql()
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/models/sql/compiler.py", line 835, in execute_sql
    cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 79, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/utils.py", line 64, in execute
    return self.cursor.execute(sql, params)
  File "/Users/mavisou/Development/network-test/network-api/venv/lib/python3.6/site-packages/django/db/backends/sqlite3/base.py", line 337, in execute
    return Database.Cursor.execute(self, query, params)
django.core.serializers.base.DeserializationError: Problem installing fixture '/Users/mavisou/Development/network-test/network-api/networkapi/fixtures/test_data.json': no such table: django_content_type
```

Was there a step missing?

File: README.md (hunk 1 above)
Comment (gideonthomas) [reply README.md]: hmm you shouldn't be seeing this if you ran `python manage.py migrate` first. That's weird

Comment (mmmavis) [reply README.md]: kk. I ran `python manage.py migrate` again and it fixed the problem.

File: README.md (hunk 2 above)
Comment (mmmavis) [reply README.md]: I ran `python manage.py migrate` again and it fixed the problem.

**Comment by mmmavis** [comment]
> The URL for fetching static content from the Network API can be customized by specifying `NETWORK_SITE_URL`. By default it uses `https://network.mofoprod.net`. NOTE: this variable must include a protocol (such as `https://`)

Is `https://network.mofoprod.net` still the right URL? If so, R+!

**Comment by gvn** [comment]
Good catch...that domain forwards, but it should be just `foundation.mozilla.org`.

**Pull request merged**